app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  class Post < ApplicationRecord
+    belongs_to :author, class_name: 'User'
+    has_many :comments
+    has_many :likes
+
+    after_save :update_posts_counter
+
+    def update_posts_counter
+      author.increment!(:post_counter)
+    end
+
+    def recent_five_comments
+      comments.last(5)
+    end
```

Comment:
- [ ] Similar to the suggestion above, kindly ensure you are fetching the comments in the correct order